src/tv2_afvd_showstyle/config-manifests.ts

```diff
@@ -231,7 +231,7 @@ export const gfxDesignTemplates: ConfigManifestEntry[] = [
 		description: '',
 		type: ConfigManifestEntryType.TABLE,
 		required: true,
-		defaultVal: [],
+		defaultVal: DEFAULT_GRAPHICS.map(val => ({ _id: '', ...val })).filter(template => template.IsDesign === true),
```

Comment:
Just write `template.isDesign` instead of `template.IsDesign === true`.
`IsDesign` already returns a boolean. No need to check the boolean ;)

Reply:
My bad, i am most certainly tired today. Fixing now!